.github/workflows/hourly-clan-sync.yml

```
@@ -40,6 +40,20 @@ jobs:
           WOM_API_KEY: ${{ secrets.WOM_API_KEY }}
           WOM_GROUP_ID: ${{ secrets.WOM_GROUP_ID }}
            
+      # Notify Discord if any sync step failed. continue-on-error means
+      # step failures don't fail the job, so we check outcomes directly.
+      - name: Notify Discord on failure
+        if: ${{ contains(steps.*.outcome, 'failure') }}
```

actionlint (1.7.12)

[error] object type "{}" cannot be filtered by object filtering `.*` since it has no object element

(expression)